assets/components/svg/svg.jsx

```diff
@@ -808,21 +855,48 @@ function SvgThankYouHeroDesktop() {
     >
       <g fill="none" fillRule="evenodd">
         <g transform="scale(1 -1) rotate(-55 -173.131 -218.706)">
-          <path d="M174.133 124.74C185.688 114.745 193 99.976 193 83.5c0-28.288-21.552-51.543-49.133-54.239C153.422 20.998 165.877 16 179.5 16c30.1 0 54.5 24.4 54.5 54.5S209.6 125 179.5 125c-1.811 0-3.602-.088-5.367-.26z" fill="#1896D7" opacity=".399" />
-          <path d="M50.647 108.866c1.273.089 2.558.134 3.853.134 30.1 0 54.5-24.4 54.5-54.5a54.256 54.256 0 0 0-10.647-32.366C126.655 24.111 149 47.696 149 76.5c0 30.1-24.4 54.5-54.5 54.5-17.98 0-33.926-8.707-43.853-22.134z" fill="#CE147C" opacity=".8" />
+          <path
```

Comment:
Is this change a formatting of the file?

Reply:
Yes